qa/suites/deepsea/tier3/upgrade/tasks/upgrade.yaml

```diff
@@ -0,0 +1,27 @@
+tasks:
+        - deepsea.preflight:
+        - deepsea.orch:
+                stage: prep
```

Comment:
humor an old man and put "allow-reboot: true" here?

Reply:
in